config/application.rb

```diff
@@ -20,46 +25,29 @@ class Application < Rails::Application
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
 
-    config.eager_load_paths << Rails.root.join('lib/')
+    # Local customisations
+    config.autoload_lib(ignore: %w(assets tasks generators))
     # Pull in folders without namespacing
-    config.eager_load_paths << Rails.root.join('app', 'models', 'areas')
-
-    # Rails generators are auto-discovered from fixed locations. The paths dont match the
-    # expected module structure. However when eager loading is enabled Zeitwerk refuses to
-    # load the code and the app fails to boot.
-    #
-    # Rails 7 has a solution to this: https://github.com/rails/rails/pull/48572
-    #
-    # For now, explicitly ignore the generator from being auto-loaded.
-    Rails.autoloaders.main.ignore(Rails.root.join('lib/generators/comparison_report/comparison_report_generator.rb'))
-
-    #config.time_zone = 'London'
-    # optional - note it can be only :utc or :local (default is :utc)
-    # HAS to be UTC for group by date to work
-    config.active_record.default_timezone = :utc
-
+    config.eager_load_paths << Rails.root.join('app/models/areas')
```

Comment:
is this necessary, there's no `app/models/areas` folder, just `area.rb`? Wondering if this is a legacy thing.

Reply:
removed